### ISSUE TITLE
exmpale/index.html not working

### ISSUE THREAD

**Issue by pgiani** [DropsOfSerenity/react-avatar-cropper#15]
can some one else confirm the exemple/index.html is not working on the current release, the file input open and nothing happens

**Comment by IlkhamGaysin** [comment]
:+1:

**Comment by lahong** [comment]
yes @pgiani  react-avatar-cropper example also doesnt work on my machine
when opened without using webpack-dev-server
but it works when opened on the browser from index.html file
any help would be appreciated
